README.md

```diff
@@ -35,16 +35,22 @@ npm install normalize-scss node-waves @ssv/core @ssv/au-core aurelia-dependency-
 ```
 
 ### Setup Plugin
-Register `au-ui` plugin with optional global configuration.
+Register `au-ui` plugin with optional global configuration and bootstrap.
 
 ```ts
-import { UiConfig, buttonType } from "@ssv/au-ui";
+import { UiConfig, UiBootstrapper, buttonType } from "@ssv/au-ui";
 
 const uiConfig: UiConfig = {
     button: { type: buttonType.raised }
 };
 // register plugin with aurelia + configure
 aurelia.use.plugin("@ssv/au-ui", uiConfig);
+
+aurelia.start().then(() => {
+    // invoke bootstrapper to initialize
+    const uiBootstrapper = aurelia.container.get(UiBootstrapper) as UiBootstrapper;
```

Comment:
true, if i find some time i might try and do a PR for them :) (its aurelia not mine)